src/budget_monitor.py

```
@@ -0,0 +1,832 @@
+
+Implementa monitoramento contínuo de custos com alertas configuráveis,
+suspensão automática e análise preditiva de budget.
+"""
+
+import json
+import time
+import threading
+import smtplib
+from pathlib import Path
+from typing import Dict, List, Any, Optional, Callable
+from datetime import datetime, timedelta
```

Ruff (0.12.2)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)